data/local_newspapers.csv

```diff
@@ -0,0 +1,52 @@
+county,localnewspapers
+Los Angeles,https://www.latimes.com

```

Comment:
does the newspaper api support https urls? Sometimes they require only http. Can you check in your colab?

Reply:
yes

Reply:
SG.